Realm/RLMSyncUser.h

```diff
@@ -134,6 +137,21 @@ NS_SWIFT_UNAVAILABLE("Use the full version of this API.");
  */
 - (void)logOut;
 
+/**
+ An optional error handler which can be set to notify the host application should
```

Comment:
"Should" in this context is technically correct, but might be confusing to non-native English speakers. `s/should the user encounter/when the user encounters/`.